.github/workflows/auto-close-stale.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     if: github.repository_owner == 'pyca'
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/stale@v8.0.0
+      - uses: actions/stale@1160a2240286f5da8ec72b1c0816ce2481aabf84 # v8.0.0
```

Comment:
do these comments with versions get updated as part of the dependabot bumps? If not we should probably just drop them since they'll swiftly become inaccurate.

Reply:
Yes they do! Dependabot also takes care of updating the comments

Reply:
Hey, it has improved significantly since the last time I looked. Thanks @diogoteles08

Reply:
Neat! We should go add comments for all of other hashes at some point.